src/client/containers/ProfilePage/BoardCardDropdown.component.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useContext } from 'react';
+import ProfileProps from './ProfileContext';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { useHistory } from 'react-router';
+import Dropdown from '../../components/Dropdown/Dropdown.component';
+
+// eslint-disable-next-line react/prop-types
+export default function DropDownMenuMyBoards({ board }) {
+  const [visible, setVisible] = useState(false);
+  const {
+    setModalVisibility,
+    setclickedBoardId,
+    setclickedBoardInfo,
+  } = useContext(ProfileProps);
+  const history = useHistory();
+  return (
+    <Dropdown
+      variant="dark"
+      onClick={() => {
+        setVisible(!visible);
+      }}
+      visible={visible}
+    >
+      <ul>
+        <li>
+          <button
+            type="button"
+            onClick={async () => {
+              // eslint-disable-next-line react/prop-types
+              await setclickedBoardId(board.id);
+              history.push('/edit-board');
+            }}
```

Comment:
@seenachacko board id for Edit page by props it will be this state " clickedBoardId "

Reply:
I just mentioned this on the other PR, rather than creating a button here and pushing to the history, I would use a `Link` from react-router that also allows you to put state on the location object, [see the documentation](https://reactrouter.com/web/api/Link/to-object).